include/boost/geometry/strategies/relate/geographic.hpp

```diff
@@ -0,0 +1,344 @@
+// Boost.Geometry
+
+// Copyright (c) 2020, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Adam Wulkiewicz, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_RELATE_GEOGRAPHIC_HPP
+#define BOOST_GEOMETRY_STRATEGIES_RELATE_GEOGRAPHIC_HPP
+
+
+// TEMP - move to strategy
+#include <boost/geometry/strategies/agnostic/point_in_box_by_side.hpp>
+#include <boost/geometry/strategies/geographic/intersection.hpp>
+#include <boost/geometry/strategies/geographic/point_in_poly_winding.hpp>
+#include <boost/geometry/strategies/spherical/point_in_point.hpp>
+#include <boost/geometry/strategies/spherical/disjoint_box_box.hpp>
+
+#include <boost/geometry/strategies/envelope/geographic.hpp>
+#include <boost/geometry/strategies/relate/services.hpp>
+#include <boost/geometry/strategies/detail.hpp>
+
+#include <boost/geometry/strategy/geographic/area.hpp>
+
+#include <boost/geometry/util/type_traits.hpp>
+
+
+namespace boost { namespace geometry
+{
+
+namespace strategies { namespace relate
+{
+
+template
+<
+    typename FormulaPolicy = strategy::andoyer,
+    // TODO: Is SeriesOrder argument needed here?
+    std::size_t SeriesOrder = strategy::default_order<FormulaPolicy>::value,
+    typename Spheroid = srs::spheroid<double>,
+    typename CalculationType = void
+>
+class geographic
```

Comment:
Here is another question. Do you think that `SeriesOrder` is needed here? The only strategies that requires it are area and segments intersection strategy. The order is used for different series in these two strategies. I'm considering removing this parameter from the umbrella strategies and sticking to the default based on `FormulaPolicy`. What do you think?

Reply:
Do you have an opinion regarding it @vissarion @barendgehrels ?